epub33/core/index.html

```diff
@@ -1340,58 +1337,55 @@ <h5>Manifest Fallbacks</h5>
-						<a href="#elemdef-package-item"><code>item</code></a> element specifies the fallback for the
-						referenced Publication Resource. The <code>fallback</code> attribute's IDREF [[XML]] value MUST
-						resolve to another <code>item</code> in the <code>manifest</code>. This fallback
-							<code>item</code> MAY itself specify another fallback <code>item</code>, and so on.</p>
+					<p>There are two cases for manifest fallbacks:</p>
 
-					<p>The ordered list of all the ID references that a Reading System can reach, starting from a given
-						item's <code>fallback</code> attribute, represents the <em>fallback chain</em> for that item.
-						The order of the resources in the fallback chain represents the EPUB Creator's preferred
-						fallback order.</p>
-
-					<p>Fallback chains MUST conform to one of the following requirements, as appropriate:</p>
-
-					<ul>
-						<li>
-							<p>For Foreign Content Documents, the chain MUST contain at least one <a>EPUB Content
-									Document</a>.</p>
-						</li>
-						<li>
-							<p>For Foreign Resources for which an EPUB Creator cannot provide an intrinsic fallback, the
-								chain MUST contain at least one <a>Core Media Type Resource</a>.</p>
-						</li>
-					</ul>
-
-					<p>Fallback chains MUST NOT contain circular or self-references to <code>item</code> elements in the
-						chain.</p>
+					<dl>
+						<dt id="spine-fallbacks">Spine Fallbacks</dt>
+						<dd>
+							<p>EPUB Creators MUST specify a fallback chain for a <a>Foreign Content Document</a> to
+								ensure that Reading Systems can always render the <a>spine</a> item. In this case, the
+								chain MUST contain at least one <a>EPUB Content Document</a>.</p>
+							<p>EPUB Creators MAY provide fallbacks for EPUB Content Documents (e.g., to provide a <a
+									href="#confreq-cd-scripted-flbk">fallback for scripted content</a>).</p>
+							<p>When a fallback chain includes more than one EPUB Content Document, EPUB Creators can use
+								the <a href="#attrdef-properties"><code>properties</code> attribute</a> to differentiate
+								the purpose of each.</p>
+						</dd>
 
-					<p>EPUB Creators MAY provide fallbacks for <a>Top-Level Content Documents</a> that are EPUB Content
-						Documents (e.g., to provide <a href="#confreq-cd-scripted-flbk">fallbacks for scripted
-							content</a>).</p>
+						<dt id="content-fallbacks">Content Fallbacks</dt>
+						<dd>
+							<div class="caution">
+								<p>The purpose for keeping content fallbacks in EPUB 3 was primarily to continue to
+									provide a means of specifying fallback images for the [[HTML]] <a
+										data-cite="html#the-img-element"><code>img</code> element</a>. As HTML now has
+									intrinsic fallback mechanism for images, the use of content fallbacks is strongly
+									discouraged. EPUB Creators should always use the intrinsic fallback capabilities of
+									[[HTML]] and [[SVG]] to provide fallback content.</p>
```

Comment:
Same comment as above. I think it would be better to move this caution to the end of the `<dd>` part.